pom.xml

```diff
@@ -98,7 +98,7 @@
         <wildfly.dist.version>11.0.0.Final</wildfly.dist.version>
         <wildflyarquilliancontainerembedded.version>2.1.0.Final</wildflyarquilliancontainerembedded.version>
 
-        <powsyblcore.version>4.7.0</powsyblcore.version>
+        <powsyblcore.version>4.10.1</powsyblcore.version>
```

Comment:
We could start using the dependencies management from powsybl-dependencies in order to simplify the dependency mangement in that pom.

Reply:
After all, not doing it:
powsybl-dependencies is not that useful here, and if at some point we want to insert powsybl-afs in powsybl-dependencies, we will need to revert this.